migration/sql-files/012-hide-email.sql

```diff
@@ -0,0 +1,3 @@
+-- Store identity provider username for the external token
+ALTER TABLE users ADD COLUMN email_hidden boolean;
```

Comment:
email_private would be better. Hidden/shown email is more UI/UX thing. Private/public is a model thing.